RussianFilter/sections/antiadblock.txt

```diff
@@ -308,7 +308,7 @@ minecraft20.ru#@#.adsBox
 ! https://github.com/AdguardTeam/AdguardFilters/issues/46073
 !+ NOT_PLATFORM(windows, mac, android)
 @@||pixel.adtelligent.com/player/adblock$domain=player.starlight.digital
-||static.adtelligent.com/static/jsvpaid.js^$redirect=nooptext,important,domain=stb.ua|novy.tv|ictv.ua|teleportal.ua
+||static.adtelligent.com/static/jsvpaid.js^$redirect=noopjs,important,domain=stb.ua|novy.tv|ictv.ua|teleportal.ua
```

Comment:
add content type. And check with Ukrainian IP how the rule works(if it is actual)